src/wob/db_book.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any
+
+from sqlalchemy import create_engine
+from sqlalchemy import Column, String
+from sqlalchemy.ext.declarative import declarative_base
+
+engine = create_engine("sqlite:///sqlalchemy.sqlite.books_info", echo=True)
```

Comment:
I think you need to figure out how to create the engine once, you shouldn't need to do this for every table you create.